Makefile

```diff
@@ -46,12 +46,18 @@ install-user-linux: kubectl-gadget-linux-amd64
 	mkdir -p ~/.local/bin/
 	cp kubectl-gadget-linux-amd64 ~/.local/bin/kubectl-gadget
 
+bpf_asset_list=$(shell find . -type f -name '*-bpf-asset.c')
+.PHONY: bpf-assets
+bpf-assets: $(bpf_asset_list:.c=.go)
+
 .PHONY: build-gadget-container
-build-gadget-container:
+build-gadget-container: bpf-assets
```

Comment:
Missing dependency on rule to build libbpf? Without it I get a `inspektor-gadget/dist/libbpf/libbpf.a: No such file or directory` when linking the gadget tracer manager.